### DISCUSSION TITLE
Edge Comments

### DISCUSSION THREAD

**Discussion by liam-edwards** [adonisjs/core#2471] [Help] [answered]
There are plenty of times I would like to include comments in my Edge templates however they shouldn't be visible to users. Laravel's Blade has a solution for this with their comment syntax: `{{-- Comment --}}`

I would like to see the same feature included in Edge.

**Comment by thetutlage** [comment, marked as answer]
Its already there in Edge and with the same syntax

**Reply by liam-edwards** [reply]
Then it's missing from the documentation. Unless I'm blind, I don't see it anywhere.

https://edge.adonisjs.com/docs/syntax-guide

**Reply by thetutlage** [reply]
![edge-comments](https://user-images.githubusercontent.com/1706381/114256938-4d13de00-99da-11eb-9e5e-04f239fb0f0b.png)

**Reply by liam-edwards** [reply]
Turns out I'm blind. Well thank you very much and sorry for wasting your time.
